src/components/ActivityIndicator/__stories__/index.stories.mdx

```diff
@@ -5,18 +5,42 @@ import {
   Meta,
   Story,
 } from '@storybook/addon-docs/blocks'
-import ActivityIndicator from '..'
-import Box from '../../Box'
+import { colors } from '../../../theme'
+import ActivityIndicator from '../index'
 
 <Meta title="Components/ActivityIndicator" component={ActivityIndicator} />
 
 # ActivityIndicator
 
-<Description>Component that indicate a loading period.</Description>
+<Description>
+  A component representing percentage as a circle which can be used to show
+  activity
+</Description>
+
+## Basic
```

Comment:
We dropped Basic title with Documentation refacto

Reply:
```suggestion
```